invenio.cfg

```diff
@@ -344,6 +344,7 @@ RDM_FILES_DEFAULT_MAX_FILE_SIZE = 50 * 10 ** 9  # 50GB
 
 JOBS_ADMINISTRATION_ENABLED = True
 
+IDENTIFIERS_SCHEMES_TO_DROP = ["SPIRES", "HAL", "OSTI", "SLAC", "PROQUEST"]
```

Comment:
this config should be inside the harvester, since it is harvester bussiness, I don't think we will change it without modifying the code so it should not be configuration variable

Reply:
not adressed yet